### PR TITLE
qdax 0.4.0 manual

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
I'm fed up with the CI errors in #8 - let's see if this works.

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin please rerender

File: recipe/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - fix-gym-dep.patch
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python ${{ python_min }}
```
Comment (wolfv) [inline recipe/recipe.yaml]:
```suggestion
    - python ${{ python_min }}.*
```

This way it should work - I tried to push to your branch but didn't work.

Comment (Tobias-Fischer) [reply recipe/recipe.yaml]: Thank you @wolfv!

**Comment by wolfv** [comment]
Hey, I checked, and it looks like you need to patch their `setup.py` file now (no more requirements.txt in the `tar.gz` file).

**Pull request merged**